hwtracer/src/decode/ykpt/mod.rs

```diff
@@ -620,9 +625,10 @@ impl<'t> YkPTBlockIterator<'t> {
                     } else {
                         let target_vaddr = self.branch_target_vaddr(&inst);
 
-                        // We can't (yet) handle setjmp in foreign code. Crash hard if we spot it.
-                        debug_assert!(setjmp_vaddr == 0 || target_vaddr != setjmp_vaddr);
-                        debug_assert!(us_setjmp_vaddr == 0 || target_vaddr != us_setjmp_vaddr);
+                        // We can't (yet) handle longjmp in unmapped code. Crash if we spot it.
```

Comment:
These should be `assert` IMHO as the system is immediately incorrect if we notice this.

Reply:
If we want them to be hard asserts we should make some effort to cache them.

Reply:
We call `dlsym()` three times per disassembly session.

Reply:
That's up to you.

Reply:
Can we say "Encountered longjmp" in the `panic`?

[I also note, with mild amusement, that `panic` does not address the caching issue you were worried with!]

Reply:
> [I also note, with mild amusement, that panic does not address the caching issue you were worried with!]

Like I said, you gave me the opportunity to be lazy :)

Reply:
Fixed in fbce6a1bd6db912e0080bd44ba607a3612694d8d